homeassistant/components/asuswrt/router.py

```diff
@@ -77,6 +227,31 @@ def update_options(self, new_options: Dict) -> bool:
         self._options.update(new_options)
         return req_reload
 
+    @property
+    def signal_device_new(self) -> str:
+        """Event specific per AsusWrt entry to signal new device."""
+        return f"{DOMAIN}-{self._unique_id}-device-new"
+
+    @property
+    def signal_device_update(self) -> str:
+        """Event specific per AsusWrt entry to signal updates in devices."""
+        return f"{DOMAIN}-{self._unique_id}-device-update"
+
+    @property
+    def unique_id(self) -> str:
```

Comment:
The router doesn't have a unique id.

Reply:
Same as comment for the related attribute